packages/forms/examples/forms/pages/autocomplete/autocomplete.page.ts

```diff
@@ -87,7 +115,7 @@ public searchItems(search: string): void {
 					this.results = [];
 				} else {
 					this.results = this.heroList.filter((hero) => {
-						return hero.name.indexOf(value) !== -1;
+						return hero.name.toUpperCase().indexOf(value.toUpperCase()) !== -1;
```

Comment:
Met het opzoeken of dit mss niet beter `toLowerCase()` moet zijn, kwam ik dit tegen: https://stackoverflow.com/questions/26877514/is-it-better-to-compare-strings-using-tolowercase-or-touppercase-in-javascript

Reply:
👍